client/auth/login.jsx

```diff
@@ -35,6 +35,27 @@ const LostPassword = React.createClass( {
 	}
 } );
 
+const SelfHostedInstructions = React.createClass( {
+
+	render: function() {
+		return (
+			<div className="auth__self-hosted-instructions">
+				<a href="#" onClick={ this.toggleSelfHostedInstructions } className="auth__self-hosted-instructions-close"><Gridicon icon="cross" size={ 24 } /></a>
+
+				<h2>{ this.translate( 'Add self-hosted site' ) }</h2>
+				<p>{ this.translate( 'By default when you sign into the WordPress.com app, you can edit blogs and sites hosted at WordPress.com' ) }</p>
+				<p>{ this.translate( 'If you\'d like to edit your self-hosted WordPress blog or site, you can do that by following these instructions:' ) }</p>
+
+				<ol>
+					<li><strong>{ this.translate( 'Install the Jetpack plugin.' ) }</strong><br /><a href="http://jetpack.me/install/">{ this.translate( 'Please follow these instructions to install Jetpack' ) }</a>.</li>
+					<li>{ this.translate( 'Connect Jetpack to WordPress.com.' ) }</li>
+					<li>{ this.translate( 'Now you can sign in to the app using the WordPress.com account Jetpack is connected to, and you can find your self-hosted site under the "My Sites" section.' ) }</li>
+				</ol>
+			</div>
```

Comment:
Should we add a call-to-action at the bottom that says "Sign In" as an alternate exit point?